### PR TITLE
Added missing descriptions to some of Image's methods and constants

### PR DESCRIPTION
**Pull request by Anutrix**
Added missing descriptions to some of Image's methods and constants.

Method description added:
 - adjust_bcs
 - compress_from_channels
 - detect_used_channels

Constant description added:
 - UsedChannels enums.

### CODE REVIEW

File: doc/classes/Image.xml [hunk 1]
```diff
@@ -700,16 +703,22 @@
 			Use BPTC compression.
 		</constant>
 		<constant name="USED_CHANNELS_L" value="0" enum="UsedChannels">
+			Only Luminance(brightness) channel is used for compression.
```
Comment (mhilbrunner) [inline doc/classes/Image.xml:706]:
```suggestion
			Only the Luminance (brightness) channel is used for compression.
```

File: doc/classes/Image.xml [hunk 2]
```diff
@@ -700,16 +703,22 @@
...
 		</constant>
 		<constant name="USED_CHANNELS_LA" value="1" enum="UsedChannels">
+			Luminance(brightness) and Alpha channels are used for compression.
```
Comment (mhilbrunner) [inline doc/classes/Image.xml:709]:
```suggestion
			Luminance (brightness) and Alpha channels are used for compression.
```

File: doc/classes/Image.xml (hunk 1 above)
Comment (Anutrix) [reply doc/classes/Image.xml]: Fixed

File: doc/classes/Image.xml (hunk 2 above)
Comment (Anutrix) [reply doc/classes/Image.xml]: Fixed.

File: doc/classes/Image.xml [hunk 3]
```diff
@@ -18,6 +18,7 @@
 			<param index="1" name="contrast" type="float" />
 			<param index="2" name="saturation" type="float" />
 			<description>
+				Adjusts the image's [param brightness], [param contrast] and [param saturation].
```
Comment (mhilbrunner) [inline doc/classes/Image.xml:21]: Would make sense to mention what the allowed value ranges are for the parameters, if its not min float / max float.

**Review by mhilbrunner — COMMENTED** [review]
Thanks for making the changes, I have some nitpicks/thoughts below. Sorry for the double review, didn't have time for an in-depth review earlier. :)

It's already an improvement, just that maybe we can make these even better. Thanks for contributing!

**Comment by YuriSizov** [comment]
This still needs to address the review comments, and it also needs a rebase now

**Comment by Mickeon** [comment]
This may no longer be necessary after https://github.com/godotengine/godot/pull/86997

One could maybe argue that the descriptions for the UsedChannels enum are better, however.

**Comment by YuriSizov** [comment]
I think we can say that it's superseded now. Further improvements can be PR'd separately. Thanks for your contribution nevertheless!

**Pull request closed without merging**